contrib/ckan/Dockerfile

```diff
@@ -1,14 +1,14 @@
-FROM python:2.7
+FROM python:3.8
 
 ENV DEBIAN_FRONTEND noninteractive
 
-ARG CKAN_VERSION=2.8.11
+ARG CKAN_VERSION=2.9.0
 
-RUN pip install -U -q pip setuptools
+RUN pip install -U -q pip "setuptools<46.0.0"
 
 RUN pip install -q -e "git+https://github.com/ckan/ckan.git@ckan-${CKAN_VERSION}#egg=ckan"
 
-ADD meta.py /src/ckan/ckan/model/meta.py
+# ADD meta.py /src/ckan/ckan/model/meta.py
```

Comment:
Lets remove this file completely if no longer useful. (It was added for [2.8](https://github.com/CodeForAfrica/openAFRICA/pull/30#discussion_r1016642964) upgrade).